packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -720,6 +720,13 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             storage,
         );
 
+        // logging hardware telemetry
+        logger.sendTelemetryEvent({
```

Comment:
I'd move it into constructor, and rename event to "DeviceSpec" or "DeviceCharacteristics"